CMakeLists.txt

```diff
@@ -747,15 +746,20 @@ if (BUILD_TESTS)
       TEST ahm_int_stress_test SOURCES AHMIntStressTest.cpp
       TEST arena_smartptr_test SOURCES ArenaSmartPtrTest.cpp
       TEST ascii_check_test SOURCES AsciiCaseInsensitiveTest.cpp
+      TEST atfork_test SOURCES AtForkTest.cpp
```

Comment:
Windows does not have `fork`.

Reply:
Fixed. A caveat with this PR: I didn't spin up a Windows docker image to see which tests just don't work on Windows.